default-python-3.rst

```diff
@@ -0,0 +1,14 @@
+.. post:: Jan 11th, 2019
+   :author: Anthony
+
+Defaulting to Python 3
+======================
+
+Starting February 1st, 2019, new projects getting started with Read the Docs
+will use Python 3 by default. While it is stil possible to configure your
+project to use Python 2.7 with :ref:`our config file <readthedocs:yaml__python>`,
+we think it's important for the ecosystem to help push projects towards adopting
+Python 3.
+
+For the time being, Python 3.6 will be our default Python 3 version, however we
+will have Python 3.7 available for testing very shortly as well.
```

Comment:
I wouldn't say "for testing" here.

Reply:
We're defaulting to 3.7 now anyways :) i'll update